docs/blog/clever-release.md

```diff
@@ -0,0 +1,20 @@
+# MakeCode Clever Integration Released!
+Just in time for the fall semester, MakeCode has integrated with [Clever](https://www.clever.com/)!
+
+Today, you can find our apps [Microsoft MakeCode Arcade](https://clever.com/library/app/makecode-arcade) and [Microsoft MakeCode for micro:bit](https://clever.com/library/app/makecode-microbit) by searching for it in the [Clever Library](https://clever.com/library).
+
+![MakeCode Arcade in the Clever Library](/docs/static/blog/clever-release/arcade-app-clever.png)
+
+Authorizing our app means your entire classroom gains instant access to MakeCode Arcade and MakeCode for micro:bit through their Clever account.
+
+Because we store projects and progress securely in the cloud, students can easily start working in the classroom and can continue at home! To do so, use the fancy new Clever button when signing in to our sites.
+
+![Clever Login Button](/docs/static/blog/clever-release/clever-login-button.png)
+
+If you can't find our apps in the Clever Library, it's possible you're in a district that allows apps for classroom use on a case-by-case basis. Please contact your district admins for help with allowing our apps within your district. For any issues you run into, please reach out to [@MSMakeCode](https://twitter.com/MSMakeCode), make a post in our forum's [Educator Lounge](https://forum.makecode.com/c/educators-lounge/24) in our forum, or [file a bug on GitHub](https://github.com/microsoft/pxt-arcade/issues/new/choose).
```

Comment:
"...district IT administrator to enable the MakeCode apps in the Clever portal"
"...post to the [Educator Lounge]..."